website/docs/docs/build/materializations.md

```diff
@@ -103,6 +104,45 @@ When using the `table` materialization, your model is rebuilt as a <Term id="tab
     * are only used in one or two downstream models, and
     * do not need to be queried directly
 
+### Materialized View
+
+The `materialized view` materialization allows the creation and maintenance of materialized views
+in the target database. This materialization makes use of the `on_configuration_change` config, which
+aligns with the incremental nature of the namesake database object. This setting tells dbt to attempt to
+make configuration changes directly to the object when possible, as opposed to completely recreating
+the object to implement the updated configuration. Using `dbt-postgres` as an example, indexes can
+be dropped and created on the materialized view without the need to recreate the materialized view itself.
+
+The `on_configuration_change` config has three settings:
```

Comment:
This section currently only pertains to materialized views (and dynamic tables for Snowflake), but could presumably apply to other materializations in the future. Should this be its own section? If so, where is the right spot?

Reply:
yeah. i think for 1.7 we should aim to have a standalone page for materialized views. but for now it can stay as a subsection of this page